### PR TITLE
Add multiple boards of the Waveshare ESP32-S3-Touch-LCD-1.46 type

### PR DESCRIPTION
**Pull request by Sail-211010**
Added several boards,Here are the boards I added this time:
Waveshare ESP32-S3-Touch-LCD-1.46
Waveshare ESP32-S3-LCD-1.46
Waveshare ESP32-S3-LCD-1.85
Waveshare ESP32-S3-Touch-LCD-1.85-Box
Waveshare ESP32-S3-LCD-1.47
Waveshare ESP32-S3-Touch-LCD-2.1
Waveshare ESP32-S3-Touch-LCD-2.8
Waveshare ESP32-S3-Relay-6CH

### CODE REVIEW

**Comment by VojtechBartoska** [comment]
Hello,

can you please change the name of the PR to more specific one?

Also please add more datials into PR description.

Usually boards are added in seperate PRs.

Did you check all already included boards?

**Comment by Sail-211010** [comment]
OK ,Here are the boards I added this time
Waveshare ESP32-S3-Touch-LCD-1.46
Waveshare ESP32-S3-LCD-1.46
Waveshare ESP32-S3-Touch-LCD-1.85-Box
Waveshare ESP32-S3-LCD-1.47
Waveshare ESP32-S3-Touch-LCD-2.1
Waveshare ESP32-S3-Touch-LCD-2.8
Waveshare ESP32-S3-Relay-6CH

**Comment by Sail-211010** [comment]
@VojtechBartoska  Hi,I have checked the uploaded file and found nothing wrong for the time being. Please contact me if you find anything wrong

**Comment by VojtechBartoska** [comment]
Thanks, we will review

File: boards.txt [hunk 1]
```diff
@@ -44367,3 +44367,1903 @@ cezerio_dev_esp32c6.menu.ZigbeeMode.rcp.build.zigbee_mode=-DZIGBEE_MODE_RCP
...
+waveshare_esp32_s3_touch_lcd_185_box.serial.disableDTR=false
+waveshare_esp32_s3_touch_lcd_185_box.serial.disableRTS=false
+
+waveshare_esp32_s3_touch_lcd_185_box.build.tarch=xtensa
+waveshare_esp32_s3_touch_lcd_185_box.build.bootloader_addr=0x0
+waveshare_esp32_s3_touch_lcd_185_box.build.target=esp32s3
+waveshare_esp32_s3_touch_lcd_185_box.build.mcu=esp32s3
+waveshare_esp32_s3_touch_lcd_185_box.build.core=esp32
+waveshare_esp32_s3_touch_lcd_185_box.build.variant=waveshare_esp32_s3_touch_lcd_185_box
+waveshare_esp32_s3_touch_lcd_185_box.build.board=WAVESHARE_ESP32_S3_TOUCH_LCD_185_Box
```
Comment (P-R-O-C-H-Y) [inline boards.txt:45111]: Please fix the board name, as its not fully uppercase
```suggestion
waveshare_esp32_s3_touch_lcd_185_box.build.board=WAVESHARE_ESP32_S3_TOUCH_LCD_185_BOX
```

**Comment by P-R-O-C-H-Y** [comment]
@Sail-211010 I see you adding a lot of boards, but the pins_arduino.h file is for some boards common, and you only specify the SPI, I2C, UART and Analog+touch pins. It would be better for your users if you can add also more pins definitions.
For example for the boards with lcd, define the LDC pins, for the relay boards add the relay channel pins etc..

**Comment by Sail-211010** [comment]
@P-R-O-C-H-Y I communicated with other engineers before and planned to define it in the example for developers to interpret the program. If there is any change in the relevant scheme in the future, I will modify the information again, and I may trouble you again. Thank you again

**Comment by P-R-O-C-H-Y** [comment]
> @P-R-O-C-H-Y I communicated with other engineers before and planned to define it in the example for developers to interpret the program. If there is any change in the relevant scheme in the future, I will modify the information again, and I may trouble you again. Thank you again

But if you define it there, you can already use the pins definition in the example and will be more useful for users. But ofc, it's your decision. Take a look for example on our [esp32-s3-usbotg board pins definition](https://github.com/espressif/arduino-esp32/blob/master/variants/esp32s3usbotg/pins_arduino.h).

**Comment by Sail-211010** [comment]
@P-R-O-C-H-Y  Due to the large size of the file, I could not edit the file online, so I deleted the file and re-uploaded it, having modified the capitalization problem you raised（Updated boards.txt）

**Comment by Sail-211010** [comment]
> > @P-R-O-C-H-Y I communicated with other engineers before and planned to define it in the example for developers to interpret the program. If there is any change in the relevant scheme in the future, I will modify the information again, and I may trouble you again. Thank you again
> 
> But if you define it there, you can already use the pins definition in the example and will be more useful for users. But ofc, it's your decision. Take a look for example on our [esp32-s3-usbotg board pins definition](https://github.com/espressif/arduino-esp32/blob/master/variants/esp32s3usbotg/pins_arduino.h).

yes,I know what you mean, I have designed this aspect on the board before, but I modified it to the current one before uploading. If necessary, I will launch a PR request next time, thank you for your trouble

**Comment by P-R-O-C-H-Y** [comment]
> > > @P-R-O-C-H-Y I communicated with other engineers before and planned to define it in the example for developers to interpret the program. If there is any change in the relevant scheme in the future, I will modify the information again, and I may trouble you again. Thank you again
> > 
> > 
> > But if you define it there, you can already use the pins definition in the example and will be more useful for users. But ofc, it's your decision. Take a look for example on our [esp32-s3-usbotg board pins definition](https://github.com/espressif/arduino-esp32/blob/master/variants/esp32s3usbotg/pins_arduino.h).
> 
> yes,I know what you mean, I have designed this aspect on the board before, but I modified it to the current one before uploading. If necessary, I will launch a PR request next time, thank you for your trouble

Would be nice to have that updated in future :) Thanks

**Comment by Sail-211010** [comment]
> > > > @P-R-O-C-H-Y I communicated with other engineers before and planned to define it in the example for developers to interpret the program. If there is any change in the relevant scheme in the future, I will modify the information again, and I may trouble you again. Thank you again
> > > 
> > > 
> > > But if you define it there, you can already use the pins definition in the example and will be more useful for users. But ofc, it's your decision. Take a look for example on our [esp32-s3-usbotg board pins definition](https://github.com/espressif/arduino-esp32/blob/master/variants/esp32s3usbotg/pins_arduino.h).
> > 
> > 
> > yes,I know what you mean, I have designed this aspect on the board before, but I modified it to the current one before uploading. If necessary, I will launch a PR request next time, thank you for your trouble
> 
> Would be nice to have that updated in future :) Thanks

Sure. I'll keep up with the situation

**Review by P-R-O-C-H-Y — APPROVED** [review]

**Comment by Sail-211010** [comment]
Hello, is there any other file error in the current progress that I need to modify

**Pull request merged**